Test/comp/Extern4.go

```diff
@@ -1,27 +1,53 @@
 package Library
```

Comment:
Why is this change in is PR?

Reply:
I goofed. I usually run the test suite on my local machine, but thought I'd let Travis take care of it. But then I forgot to check the status before doing the pull request. It should have been part of the original pull request.